### PR TITLE
integrate icons and alerts component

### PR DESCRIPTION
**Pull request by kalidiagne**
a simple icon integration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -124,5 +124,10 @@
     "> 1%",
     "last 2 versions",
     "not ie <= 8"
-  ]
+  ],
+  "dependencies": {
+    "html-loader": "^0.4.3",
```
Comment (cristijora) [inline package.json:129]: These should be in devDependencies

**Comment by cristijora** [comment]
Thanks a lot!
I was pretty busy in the last couple of weeks and didn't find time to continue working on this so any help is much appreciated! Looks good at first sight except some `dependencies` which have to be `devDependencies`. Will look at the PR more in depth later on

**Comment by kalidiagne** [comment]
I hope i can be helpful, i added alert component.

**Comment by cristijora** [comment]
Would be very grateful if you open a new PR of each new component because it's hard to analyze changes for several components at once. Keep this one as it is but if you want to add a new component, please make a new PR

**Comment by kalidiagne** [comment]
ok @cristijora , i released new PR... now i'm going to keep in mind it.

**Comment by cristijora** [comment]
I will merge these PRs for now even though there are some changes I would require but because they are linked between them its easier if I merge them and then make my changes on top of that. 
Some guidelines for further PRs

- 1 PR for 1 Component
- Install everything under devDependencies (html-loader for example)
- Make sure the component has docs and the implementation is at least functional/complete
- I will make a dev branch and the target branch for PRs will be the dev branch

**Pull request merged**